<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1010</width>
    <height>854</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>outsideSort</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>971</width>
      <height>711</height>
     </rect>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>760</y>
      <width>751</width>
      <height>31</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>760</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>进 度</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1010</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>数据</string>
    </property>
    <addaction name="actDataForm"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>排序</string>
    </property>
    <addaction name="actSort"/>
    <addaction name="actInside"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>打开</string>
    </property>
    <addaction name="actOpenFile"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>清空</string>
    </property>
    <addaction name="actClear"/>
   </widget>
   <addaction name="menu_3"/>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_4"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actDataForm">
   <property name="text">
    <string>新建数据文件</string>
   </property>
   <property name="toolTip">
    <string>随机数据文件形成</string>
   </property>
  </action>
  <action name="actSort">
   <property name="text">
    <string>竞赛树生成归并段</string>
   </property>
   <property name="toolTip">
    <string>选择要排序的文件</string>
   </property>
  </action>
  <action name="actOpenFile">
   <property name="text">
    <string>选择要打开的文件</string>
   </property>
   <property name="toolTip">
    <string>选择要打开的文件</string>
   </property>
  </action>
  <action name="actClear">
   <property name="text">
    <string>清空</string>
   </property>
   <property name="toolTip">
    <string>清空显示框和进度条</string>
   </property>
  </action>
  <action name="actInside">
   <property name="text">
    <string>内部排序生成归并段</string>
   </property>
   <property name="toolTip">
    <string>内部排序生成归并段</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
